components/x-teaser/src/v2/components/Image/index.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable jsx-a11y/alt-text */
+
+import { h } from '@financial-times/x-engine';
+import c from 'classnames';
+
+// Transparent PNG with 16:9 dimensions
+const placeholderSrc = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAKAAAABaCAQAAACVz5XZAAAAhklEQVR42u3QMQ0AAAwDoNW/6Yno1QQkkKMSBQIFChSIQIECBSJQoECBCBQoUCACBQoUiECBAgUiUKBAgQgUKFAgAgUKFIhAgQIFIlCgQIECEShQoEAEChQoEIECBQpEoECBAhEoUKBABAoUKBCBAgUKRKBAgQIRKFCgQAQKFChQoAKBApc9gMUAW29OZusAAAAASUVORK5CYII=';
+
+const Image = ({ src, isLazyLoaded, lazyLoadClassName, className }) => {
+	const fullClassName = c(className, lazyLoadClassName);
+	const actualSrc = src || placeholderSrc;
+	const commonProps = {
+		className: fullClassName,
+		alt: 'Article image',
```

Comment:
Is this a default alt attribute for when one hasn't been provided? If so better to default to an empty string than have something with no meaning (but omitting the alt attribute makes most screen readers read out the url).

Reply:
fixed